fiware-region-sanity-tests/tests/fiware_region_with_networks_tests.py

```diff
@@ -33,6 +31,12 @@
 from commons.template_utils import replace_template_properties
 import re
 import json
+import uuid
+
+
+def _build_path_resource(path_resource):
+    """Build url path with a transactionId param"""
+    return path_resource + '?TransactionId=' + str(uuid.uuid1())
```

Comment:
Here's where you can use the "txIdHeader" constant suggested in a previous comment